src/components/SplitButton/__stories__/SplitButtonExampleDialog.jsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
```

Comment:
the documentation doesn't show this code, so it makes it hard for consumers to understand how to use the component.

See how it looks like
![image](https://github.com/mondaycom/monday-ui-react-core/assets/6093192/e6561fc8-b10f-4619-b941-aaa25f647bab)

I suggest to kill this file and make it inline in the code example instead

Reply:
hmm good point.
will do

Reply:
+1 :)